blocks/quote/quote.css

```diff
@@ -0,0 +1,116 @@
+body {
+  background-color: #2B2B2B;
+}
+
+.quote-container {
+  padding-top: 0;
+  width: 100vw;
+}
+
+.quote {
+  margin: 0;
+  width: 100%;
+
+  ul {
+    list-style-type: none;
+    padding: 0;
+    margin: 0 auto;
+    width: 1440px;
+    max-width: 96vw;
+    
+    li {
+      padding: 0.5rem;
+    }
+  }
+
+  .cards-card-body {
+    box-sizing: border-box;
+    padding: 3rem 0.5rem 1.5rem 1rem;
+    font-family: var(--calcite-sans-family);
+    width: 1440px;
+    max-width: 96vw;
+
+    .quote-icon {
+      --calcite-ui-icon-color: #009AF2;
```

Comment:
Color should be `var(--calcite-ui-brand)`
I've added the Calcite CSS file in another pull request.